pkg/plan/resource/deb.go

```diff
@@ -23,6 +24,10 @@ func (d *Deb) State() plan.State {
 	return toState(d)
 }
 
+func isLowerRevision(v1, v2 *Deb) bool {
+	return strings.Compare(v1.Suffix, v2.Suffix) < 0
```

Comment:
Does this work for the sorts of version strings we might encounter?
I don't know much about Deb versions but would we get, say, 1.10 vs 1.9?

Reply:
Yes I agree that for such case it will consider 1.10 coming before 1.9
I'll recheck this function again. Thank you 👍

Reply:
Looking around at the code, do we want to do this check at all?
Why not just give the version to `apt` and let it decide what to do?

Reply:
I'm sorry Bryan, didn't see your comment before pushing! I don't recall the bug Yiannis and I were facing when we did this. I think we can give it another go.